### DISCUSSION TITLE
LDM Notes for October 11th, 2023

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#7594] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2023/LDM-2023-10-11.md

### Agenda

- C# spec update
- Collection expressions

**Comment by TonyValenti** [comment]
Do you try to get every version of c# standardized or do you skip over interim versions?

**Reply by 333fred** [reply]
We skipped over the minor versions of C# 7. There will be no separate specs for 7.1/2/3, as an example. @BillWagner can clarify, but I believe the committee is planning on doing a spec for each of 8/9/10/11/12+.

**Reply by BillWagner** [reply]
We added all the changes for the point releases of C# 7 into the C# 7 standard for the following reasons:

1. Some of the point releases modify features added in C# 7. To create a standard for each point release meant extra work for very little value, when weigh against trying to catch up to the implementation.
2. C# 7.3 is that last version supported by .NET Framework. We wanted to release a standard that matched that version.

Going forward, we focus on each release. It's the easiest way for us to stay organized and test behavior. We're currently working on C# 8. We may choose not to submit each release to ECMA for approval. It's more likely that we will submit each for approval, but that's a decision we'll make on each version. Regardless, we'll tag the branches so interested readers can find each major version.

**Reply by jnm2** [reply]
Should the C# 7 spec be labeled as the 7.3 spec to help avoid confusion since it's not the 7.0 spec?
